--- cimv216.mof	2007-08-30 16:19:12.000000000 -0700
+++ cimv216-new.mof	2007-12-05 12:45:56.000000000 -0800
@@ -505,7 +505,7 @@
 #pragma include ("Policy/CIM_PolicySetValidityPeriod.mof")
 #pragma include ("Policy/CIM_PublicPrivateKeyAuthentication.mof")
 #pragma include ("Policy/CIM_SharedSecretAuthentication.mof")
-#pragma include ("Security/CIM_SecurityIndication.mof")
+//#pragma include ("Security/CIM_SecurityIndication.mof")
 #pragma include ("Support/PRS_Activity.mof")
 #pragma include ("Support/PRS_ActivityResource.mof")
 #pragma include ("Support/PRS_ActivityTransaction.mof")
@@ -726,8 +726,8 @@
 #pragma include ("Policy/CIM_PolicyActionStructure.mof")
 #pragma include ("Policy/CIM_PolicyConditionInPolicyCondition.mof")
 #pragma include ("Policy/CIM_PolicyConditionInPolicyRule.mof")
-#pragma include ("Security/CIM_IPNetworkSecurityIndication.mof")
-#pragma include ("Security/CIM_IPPacketFilterIndication.mof")
+//#pragma include ("Security/CIM_IPNetworkSecurityIndication.mof")
+//#pragma include ("Security/CIM_IPPacketFilterIndication.mof")
 #pragma include ("Support/PRS_ActivityContact.mof")
 #pragma include ("Support/PRS_AdminAssociation.mof")
 #pragma include ("Support/PRS_AdministrativeContact.mof")
